src/language/providers/utils.ts

```diff
@@ -276,14 +276,15 @@ export function checkMultiLineTag(
     while (currentText.trim() === '' || !currentText.includes('<')) {
       --currentLine
       currentText = document.lineAt(currentLine).text
-      if (!currentText.endsWith('/>')) {
-        break
-      }
+      //      if (!currentText.endsWith('/>')) {
+      //        break
+      //      }
```

Comment:
Remove commented code.

Reply:
I wasn't sure if I needed, I was going to remove after testing, but forgot. It's removed now.